### PR TITLE
HAA-433 : display nomis emailid

### PR DESCRIPTION
**Pull request by ravmoj**
Display Nomis emailId on beta search

### CODE REVIEW

File: backend/routes/searchApiFactory.js [hunk 1]
```diff
@@ -64,14 +64,9 @@ function searchApiFacade(prisonApi, oauthApi, nomisUsersAndRolesApi, manageUsers
       }
 
     // now augment with auth email addresses
```
Comment (brightonsbox) [inline backend/routes/searchApiFactory.js]: Can remove this comment now - not doing this anymore

File: backend/controllers/searchWithFilter.js [hunk 2]
```diff
@@ -47,6 +47,7 @@ const searchFactory = (
 
     const results = searchResults.map((user) => ({
       usernameAndEmail: mapUsernameAndEmail(user),
+      nomisEmailId: user.email,
```
[the remark this answers is not in the document]
Comment (ravmoj) [reply backend/controllers/searchWithFilter.js:50]: usernameAndEmail is used in 'externalSearchResults.njk' and this user object is changed to 'user name  / email id' in function mapUsernameAndEmail(user)

[the remark this answers is not in the document]
Comment (brightonsbox) [reply backend/controllers/searchWithFilter.js:50]: But aren't we rendering `searchWithFilter.njk` here?

[the remark this answers is not in the document]
Comment (brightonsbox) [reply backend/controllers/searchWithFilter.js:50]: Also I noticed that line 54 feedbackSource isn't used so could remove that while we're here

File: backend/controllers/userDetails.js [hunk 3]
```diff
@@ -23,6 +23,7 @@ const userDetailsFactory = (
     const searchTitle = req.session.searchTitle ? req.session.searchTitle : defaultSearchTitle
     const searchUrl = req.session.searchUrl ? req.session.searchUrl : defaultSearchUrl
     const searchResultsUrl = req.session.searchResultsUrl ? req.session.searchResultsUrl : `${searchUrl}/results`
+    const nomisEmailId = req.query.nomisEmail !== 'null' && req.query.searchType === 'beta' ? req.query.nomisEmail : ''
```
[the remark this answers is not in the document]
Comment (ravmoj) [reply backend/controllers/userDetails.js]: Updated to show nomis email id on User Details page for old and beta searches

File: backend/controllers/userDetails.js [hunk 4]
```diff
@@ -35,7 +36,11 @@ const userDetailsFactory = (
       searchTitle,
       searchResultsUrl,
       searchUrl,
-      staff: { ...user, name: `${user.firstName} ${user.lastName}` },
+      staff: {
```
[the remark this answers is not in the document]
Comment (ravmoj) [reply backend/controllers/userDetails.js]: reverted

File: views/searchWithFilter.njk [hunk 5]
```diff
@@ -169,7 +169,7 @@
                             <div class="govuk-grid-column-one-half">
                               <span class="govuk-visually-hidden">Status</span>
                                 {{ govukTag({
-                                  text: u.status,
+                                  text: "Inactive",
```
Comment (Mjwillis) [inline views/searchWithFilter.njk:172]: why have you changed that back?

**Review by Mjwillis — CHANGES_REQUESTED** [review]
Over-written earlier change

File: views/searchWithFilter.njk (hunk 5 above)
Comment (ravmoj) [reply views/searchWithFilter.njk]: Merge issue, reverted to main branch changes

**Review by Mjwillis — APPROVED** [review]
LGTM

**Pull request merged**